fe/fe-core/src/main/java/com/starrocks/StarRocksFE.java

```diff
@@ -195,6 +204,89 @@ public static void start(String starRocksDir, String pidDir, String[] args) {
         System.exit(0);
     }
 
+    private static void handleGracefulExit(QeService qeService) {
+        Signal.handle(new Signal("USR1"), sig -> {
```

Comment:
signal USER1 -> signal 10
signal SIGTERM -> signal 15

Reply:
SIGTERM is used for normal exit, so we have to use another signal for graceful exit